.gitignore

```diff
@@ -1,5 +1,4 @@
 /node_modules/
 npm-debug.log
-/lib/
```

Comment:
Removing this from the ignore file seems bad, it means all the generated code is now in source control, any reason why you've done this?